python/starter_kit/README.md

```diff
@@ -8,3 +8,17 @@ Dans cette fiche, nous allons voir quelles sont les étapes pour bien démarrer
 ## Table des matières
 
 - [Comment bien commencer avec python](fr/START_WELL.md)
+
+
+# Starting off right with Python
```

Comment:
```suggestion
# Starting Off Right with Python
```
In English, titles usually have capittal letters for each word that's not "in", "with", etc.